ostelco-ios-client/AppDelegate.swift

```diff
@@ -122,10 +122,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if EmailLinkManager.isSignInLink(incomingURL) {
             EmailLinkManager.signInWithLink(incomingURL)
                 .then {
-                    UserManager.shared.getDestinationFromContext()
+                    self.rootCoordinator.determineDestination()
                 }
                 .done { [weak self] destination in
-                    self?.rootCoordinator.navigate(to: destination, from: nil)
+                    self?.rootCoordinator.navigate(to: destination, from: nil, animated: true)
```

Comment:
So the coordinator figured out where should be next which is handy to keep up with, but here we're relying on the AppDelegate to actually go there next. In theory the root coordinator already knows we need to go somewhere and could direct that without the app delegate asking it to?

Reply:
Yeah that makes sense.